src/Microsoft.DncEng.SecretManager/PasswordGenerator.cs

```diff
@@ -0,0 +1,34 @@
+using System.Security.Cryptography;
+using System.Text;
+
+public static class PasswordGenerator
+{
+    public static string GenerateRandomPassword(int length)
```

Comment:
I think this algorithm might not be complex enough for github passwords. Completely random alphanumeric is plenty complex, but I think github might yell if it doesn't have a symbol or something. I don't want to put symbols in the sql passwords though, so we probably need to make this code able to generate both.

Reply:
I updated it to be able to generates passwords with special characters from ! to ~. I didn't include space, it's valid for password, but it could make troubles with copy/paste.